.gitignore

```diff
@@ -7,7 +7,7 @@
 .svn
 *~
 manifest.appcache
-
+/.idea
```

Comment:
I'd probably remove the .idea from the .gitignore since, as Johan says, we don't really own it.

FYI, you can add things to gaia/.git/info/exclude and it'll work like gitignore for your local clone, but won't get checked in.